backend/routes/drops_intake.py

```
@@ -353,100 +292,60 @@ async def get_processed_drops(
+            "created_at": created_at.isoformat() if hasattr(created_at, "isoformat") else str(created_at),
         }
 
-    except Exception as e:
-        raise HTTPException(status_code=500, detail=f"Database error: {str(e)}")
-
+    except HTTPException:
+        raise
+    except Exception as exc:
+        if conn:
+            conn.rollback()
+        raise HTTPException(status_code=500, detail=f"Intake error: {str(exc)}") from exc
```

Ruff (0.15.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)